openmdao/docs/feature_reference/building_blocks/drivers/pyoptsparse_driver.rst

```diff
@@ -3,6 +3,71 @@
 pyoptsparse Driver
 ******************
 
+The pyOptSparseDriver wraps the optimizer package pyOptSparse, which provides a common interface for 11 optimizers, some of which
+are included in the package (e.g., SLSQP and NSGA2), and some of which are commercial products that must be obtained from their
+respective authors (e.g. SNOPT). The pyOptSparse package is based off of pyOpt, but it adds support for sparse specification of
+constraint jacobians. Most of the sparsity features are only applicable when using the SNOPT optimizer.
+
+.. note::
+    The pyOptSparse package does not come included with the OpenMDAO installation. It is a separate optional package that can be obtained
+    from  mdolab_.
+
+In this simple example, we use the SLSQP optimizer to minimize the objective of SellarDerivativesGrouped.
+
+.. embed-test::
```

Comment:
I would like to include the import statements inside the test. The user can't easily run the example otherwise. 

How would they know where to find SellarDerivativesGrouped?

Reply:
We don't have imports in any of our feature docs, and this is a big change to how the directives currently operate. I think it would be beneficial, but it's a separate story (and maybe challenging) that shouldn't hold up this pull request. Story should be to automatically figure out what imports need to be added, and add them for the `embed_test` and `embed_code` directives.

Alternatively, or as a workaround, I can embed the SellarDerivativesGroup code.